src/voxel.py

```diff
@@ -105,6 +105,8 @@ def __init__(self):
     def _initialise_data(self):
         # Our scene data
         self._data = self.blank_data()
+        # Create empty selection
+        self._selection = []
```

Comment:
It's called `add` because there is no real order in a set. I'll change it while merging.